ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -839,6 +1098,33 @@ private BlobReadOptions getDeletedBlobReadOptions(IndexValue value, StoreKey key
     return readOptions;
   }
 
+  /**
+   * Gets {@link BlobReadOptions} for a undeleted blob.
+   * @param value the {@link IndexValue} of the delete index entry for the blob.
+   * @param key the {@link StoreKey} for which {@code value} is the delete {@link IndexValue}
```

Comment:
please update comment for `value` and `key` parameters (change delete to undelete)